src/components/NotificationBanner/NotificationBanner.stories.tsx

```diff
@@ -29,9 +29,6 @@ import {
 export default {
   title: 'Components/Notification/NotificationBanner',
   component: NotificationBanner,
-  parameters: {
-    layout: 'fullscreen',
-  },
 };
 
 export const Base = (args: NotificationBannerProps) => (
```

Comment:
@connor-baer I was also wondering if we should remove the args in this case. I understand they are replacing knobs but I don't think they are helpful in this case, since this component only accepts a child (a ReactNode).

Or should we keep it just to make sure that we keep consistency across all components?

Reply:
You're right, the args are not very helpful in this case.